js/src/components/google-account-card/connected-google-account-card.js

```diff
@@ -1,14 +1,58 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
 /**
  * Internal dependencies
  */
+import { glaData } from '.~/constants';
 import AccountCard, { APPEARANCE } from '.~/components/account-card';
+import ConnectedIconLabel from '.~/components/connected-icon-label';
+import useGoogleAuthorization from '.~/hooks/useGoogleAuthorization';
+import useDispatchCoreNotices from '.~/hooks/useDispatchCoreNotices';
+import Section from '.~/wcdl/section';
+import AppButton from '../app-button';
 
 export default function ConnectedGoogleAccountCard( { googleAccount } ) {
+	const pageName = glaData.mcSetupComplete ? 'reconnect' : 'setup-mc';
+	const { createNotice } = useDispatchCoreNotices();
+	const [ fetchGoogleConnect, { loading, data } ] = useGoogleAuthorization(
+		pageName
+	);
+
+	const handleConnect = async () => {
+		try {
+			const { url } = await fetchGoogleConnect();
+			window.location.href = url;
+		} catch ( error ) {
+			createNotice(
+				'error',
+				__(
+					'Unable to connect your Google account. Please try again later.',
+					'google-listings-and-ads'
+				)
+			);
+		}
+	};
```

Comment:
Note: this is the same as the following in `AuthorizeGoogleAccountCard `, I plan to refactor this in a separate smaller PR.

https://github.com/woocommerce/google-listings-and-ads/blob/0cad7cd8defcfbd433f91ea963b12820cba41f24/js/src/components/google-account-card/authorize-google-account-card.js#L45-L58

Reply:
This is addressed in https://github.com/woocommerce/google-listings-and-ads/pull/1074.